solo-chains/runtime/starlight/src/genesis_config_presets.rs

```diff
@@ -317,7 +317,7 @@ fn starlight_testnet_genesis(
             "config": runtime_parachains::configuration::HostConfiguration {
                 scheduler_params: SchedulerParams {
                     max_validators_per_core: Some(1),
-                    num_cores: para_ids.len() as u32,
+                    num_cores: 4,
```

Comment:
why this?

Reply:
To support registering new chains in zombienet. Without this, chains 2000 and 2001 work, but when we register 2002 it won't have a core. I set it to 4 because we have 4 validators.

Reply:
alright, if it does not break anything else..

Reply:
fixed

Reply:
Why dont we control the `num_cores` from withing zombienet itself?